CIP-0036/test-vector.md

```diff
@@ -1,110 +1,145 @@
-  3: '0xe0ae3a0a7aeda4aea522e74e4fe36759fca80789a613a58a4364f6ecef',
-  4: 1234,
-  5: 0
-},
+"61284": {
+  "1": [["0x0036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a0", 1]],
+  "2": "0x5820e3cd2404c84de65f96918f18d5b445bcb933a7cda18eeded7945dd191e432369",
+  "3": "0x004777561e7d9ec112ec307572faec1aff61ff0cfed68df4cd5c847f1872b617657881e30ad17c46e4010c9cb3ebb2440653a34d32219c83e9",
+  "4": 1234,
+  "5": 0
+}
 ```
 
+### CBOR Encoding
 
-Metadata (CBOR encoding)
-
-Legacy:
+Legacy CIP-15 version:
 ```
-a119ef64a4015820a6a3c0447aeb9cc54cf6422ba32b294e5e1c3ef6d782f2acff4a70694c4d166302582086870efc99c453a873a16492ce87738ec79a0ebd064379a62e2c9cf4e119219e03581de0ae3a0a7aeda4aea522e74e4fe36759fca80789a613a58a4364f6ecef041904d2
+a119ef64a40158200036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a0025820e3cd2404c84de65f96918f18d5b445bcb933a7cda18eeded7945dd191e43236903581de072b617657881e30ad17c46e4010c9cb3ebb2440653a34d32219c83e9041904d2
 ```
 
-New:
+CIP-36 version:
 ```
-a119ef64a50182825820a6a3c0447aeb9cc54cf6422ba32b294e5e1c3ef6d782f2acff4a70694c4d16630182582000588e8e1d18cba576a4d35758069fe94e53f638b6faf7c07b8abd2bc5c5cdee0302582086870efc99c453a873a16492ce87738ec79a0ebd064379a62e2c9cf4e119219e03581de0ae3a0a7aeda4aea522e74e4fe36759fca80789a613a58a4364f6ecef041904d20500
+a119ef64a501818258200036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a001025820e3cd2404c84de65f96918f18d5b445bcb933a7cda18eeded7945dd191e432369035839004777561e7d9ec112ec307572faec1aff61ff0cfed68df4cd5c847f1872b617657881e30ad17c46e4010c9cb3ebb2440653a34d32219c83e9041904d20500
 ```
 
-Blake2b-256 hash of metadata
+### Blake2b-256 Hash
 
-Legacy:
+Legacy CIP-15 version:
 ```
-872bcb4a9e2b110a06fd5de04be5924b6c659c28a1665ecc75def13ebca6dfd8
+9946e71b5f6c16150cf431910a0f7dbb8084a992577847802e60d32becb3d6be
 ```
 
-New:
+CIP-36 version:
 ```
-5bc0681f173efd76e1989037a3694b8a7abea22053f5940cbb5cfcdf721007d7
+3110fbad72589a80de7fc174310e92dac35bbfece1690c2dce53c2235a9776fa
 ```
 
-### Output
+## Metadata Example with Witness
 
-Legacy:
+Legacy CIP-15 version:
 ```javascript
 {
-  61284: {
-    1: '0x0036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a0',
-    2: '0x1c5d88aa573da97e5a4667e0f7c4a9c6a3d848934c3b0a5b9296b401540f2aef',
-    3: '0xe0ae3a0a7aeda4aea522e74e4fe36759fca80789a613a58a4364f6ecef',
-    4: 1234,
-  },
-  61285: {
-    1: '0xcf8e34db193edc930faf64626a0a759dd9ce654874d4b6f255dc2aa8f52313b6dbb9aa1b162b43ed8946668edca920dd34f5a51a14130814fdc86adb6218b501'
+  "61284": {
+    "1": "0x0036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a0",
+    "2": "0x5820e3cd2404c84de65f96918f18d5b445bcb933a7cda18eeded7945dd191e432369",
+    "3": "0xe072b617657881e30ad17c46e4010c9cb3ebb2440653a34d32219c83e9",
+    "4": 1234
+  }
+  "61285": {
+    "1": "0xa9ec8735804c6c4c5c4a02e9589c65508ec7060063b2d7dbeba82d1cbfa1b8be6b457f95d4ead5e8b454b989624fa44e0b89a64d089fdc0a6a1268fef4876d0f" 
   }
 }
 ```
 
-New:
+CIP-36 version:
```

Comment:
the provided example is not valid. because the vote-public-key is actually the stake-public-key. as we derive the vote-public-key now from path 1694H/1815H/0H/0/0, it can't be the same as the stake-public-key.

maybe also use an example like:
```json
{
  "type": "PaymentVerificationKeyShelley_ed25519",
  "description": "Governance Verification Key",
  "cborHex": "5820423fa841abf9f7fa8dfa10dacdb6737b27fdb0d9bcd9b95d48cabb53047ab769"
}
```
bech32: "gov_vk1ggl6ssdtl8ml4r06zrdvmdnn0vnlmvxehnvmjh2ge2a4xpr6ka5suatdx2"

for the vote-public-key ?

Reply:
> The catalyst private signing key is a bit misleading without the public key

I had just reused the voting private key from previous test vectors, good idea to add in the public side.

> i am not sure why the payment vkey/skey at the top are included. What are they there for?

As for vkey/skey, you're right there are not needed, I was leaning towards including everything to aid transparency and recreation - this was also done in the [CIP-62](https://github.com/cardano-foundation/CIPs/pull/296) test vectors. I will ask other's opinions on this as I think this is a good point.